### ISSUE TITLE
`valid-expect` should not error on `expect.poll` with options

### ISSUE THREAD

**Issue by DesselBane** [vitest-dev/eslint-plugin-vitest#932]
Currently code like this will trigger a rule error

```ts
await expect.poll(() => 2, { timeout: 5000 }).toBe(2)
await expect.poll(() => 2, { interval: 5000 }).toBe(2)
await expect.poll(() => 2, { message: "foo" }).toBe(2)
await expect.poll(() => 2, { timeout: 5000, interval: 5000, message: "foo" }).toBe(2)
```

## Expected Behavior

No error.

I would argue the rule should not trigger on `expect.poll` at all since it does not accept multiple arguments in the first place, if that isnt possible it should at least allow the options from `expect.poll`
